src/test/java/org/ethereum/beacon/discovery/Attack.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.ethereum.beacon.discovery;
+
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.ethereum.beacon.discovery.packet.Header;
+import org.ethereum.beacon.discovery.packet.OrdinaryMessagePacket;
+import org.ethereum.beacon.discovery.packet.RawPacket;
+import org.ethereum.beacon.discovery.packet.impl.RawPacketImpl;
+import org.ethereum.beacon.discovery.type.Bytes12;
+import org.ethereum.beacon.discovery.type.Bytes16;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public class Attack {
+  @Test
+  @Disabled
+  void dos() throws Exception {
```

Comment:
Not sure we should commit this - it's disabled so won't run and is one of those bits of code that's just likely to rot and wind up not working next time we actually go to run it anyway.

Reply:
Makes sense. Deleted.